handsontable/src/plugins/multiColumnSorting/__tests__/multiColumnSorting.spec.js

```diff
@@ -13,9 +13,7 @@ describe('MultiColumnSorting', () => {
         throw Error('Please check the test scenario. The header doesn\'t exist.');
       }
 
-      $spanInsideHeader.simulate('mousedown');
-      $spanInsideHeader.simulate('mouseup');
-      $spanInsideHeader.simulate('click');
+      simulateClick($spanInsideHeader[0]);
```

Comment:
```suggestion
      simulateClick($spanInsideHeader);
```
AFAIK the helper accepts the jQuery element instance as well.

Reply:
Changed within https://github.com/handsontable/handsontable/pull/10210/commits/5df751b593920ca0f2e63f09e7a9522ec8d0425c.